dicom-archive/DICOM/Fields.pm

```diff
@@ -1,9 +1,62 @@
-# Definitions of fields of DICOM headers.
-# Andrew Crabb (ahc@jhu.edu), May 2002.
-# $Id: Fields.pm 4 2007-12-11 20:21:51Z jharlap $
-
 package DICOM::Fields;
 
+=pod
+
+=head1 NAME
+
+DICOM::Fields -- Definitions of fields of DICOM headers.
+
+=head1 SYNOPSIS
+
+  use DICOM::Element;
+  use DICOM::Fields;	# Standard header definitions.
+  use DICOM::Private;	# Private or custom definitions.
+
+  # Initialize dictionary.
+  # Read the contents of the DICOM dictionary into a hash by group and element.
+  # dicom_private is read after dicom_fields, so overrides common fields.
+  BEGIN {
```

Comment:
Not a big fan of this BEGIN block. I think this should be refactored (see review of #232 ) at a later time.

Reply:
agreed